README.md

```diff
@@ -162,3 +162,40 @@ The script will create a file called `linkidtopatid.csv` with the mapping of LIN
 Copyright 2020 The MITRE Corporation.
 
 Approved for Public Release; Distribution Unlimited. Case Number 19-2008
+
+# Data Owner Tools Executables
+Fork of https://github.com/mitre/data-owner-tools. All python files should conform to existing command line documentation. Command line and GUI tools should be interoperable. Executables built using pyinstaller version 4.2. Only Windows executables availible in releases.
```

Comment:
Please remove the text about this being a fork. When we merge it in, it will just be in the repo.